frodo.js

```diff
@@ -5,77 +5,72 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!